app/src/MyComponent.js

```diff
@@ -12,86 +12,81 @@ const {
 export default ({ drizzle, drizzleState }) => { // destructure drizzle and drizzleState from props
   return (
     <div className="App">
-    <div>
-      <img src={logo} alt="drizzle-logo" />
-      <h1>Drizzle Examples</h1>
-      <p>Examples of how to get started with Drizzle in various situations.</p>
-    </div>
+      <div>
+        <img src={logo} alt="drizzle-logo" />
+        <h1>Drizzle Examples</h1>
+        <p>Examples of how to get started with Drizzle in various situations.</p>
+      </div>
 
-    <div className="section">
-      <h2>Active Account</h2>
-      <AccountData drizzle={drizzle} drizzleState={drizzleState} accountIndex={0} units="ether" precision={3} />
-    </div>
+      <div className="section">
+        <h2>Active Account</h2>
+        <AccountData drizzle={drizzle} drizzleState={drizzleState} accountIndex={0} units="ether" precision={3} />
+      </div>
 
-    <div className="section">
-      <h2>SimpleStorage</h2>
-      <p>
-        This shows a simple ContractData component with no arguments, along with
-        a form to set its value.
-      </p>
-      <p>
-        <strong>Stored Value: </strong>
-        <ContractData drizzle={drizzle} drizzleState={drizzleState} contract="SimpleStorage" method="storedData" />
-      </p>
-      <ContractForm drizzle={drizzle} contract="SimpleStorage" method="set" />
-    </div>
+      <div className="section">
+        <h2>SimpleStorage</h2>
+        <p>
+          This shows a simple ContractData component with no arguments, along with a form to set its value.
```

Comment:
It looks like you un-wrapped this block of text into one line. This makes it difficult to read without scrolling horizontally or setting your editor to auto-wrap (in which case the indentation would be off).

To auto-format for these errors, run the code through [Prettier](https://prettier.io/).

[Example](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBLAtgBwgJxgAgCU4BDMAgMxwnXwB0QdTyGBuOqDbPfYfKOAHcAwtHgAPGKKzQEMAM74Avvio16IAAIATHKgBe+gDZwA9EzIwAtJBkDY8thy64CRiAHMIq6rQYA6U3cvf0woDycoDkgoeQJgDnx8AEEwSABXWAAREhgSABpE-FFYHEscvMKoJJKYMvIAMVx0KpUAXn4hWrhJaWx7BXYo4Z7uAm04ChJ0owIACj5dA2M4fPwlwxMAZTz4ZQBKfDaAPl58U1N1uDicdPJ0pnW9Tbh8EihtJ+Xt3de1Wkw1Ewjmq+CYMAe1TmRSSAB5tKgAG74MBGEjyeQAORI6DgbQYyUwmAYxxhSXw8KRpNB5PJsIwHnw8hwYDawGCEBUJFm+JAGxWVg5DHO1NptNhAAsAIzHLLPFb4ACi4hxmBM8lhpmlorFcMwx2VqvV+AgFHwEoggnwMG8HjgBDiJDwcE+glQMAl+Dl31eqGqiKdqAg6UU8nd6VyQdi-k1+rJcNMCMR1PjFKTKLRGOxuN58jg5CjJNTcIlACZjqkYEjXqkMrBNWWdbrYbXg9lciQvi82fyTCpe3Adrk8cAB0P4Fy0m2YABJD49NkABhUmXd8l59olcBwwsB+dQYegbIAzCpTE26YmqRxU5Tkaj0VicXiGHmC9AizSxZLy1suD9cBIO0G3LYsKTjL9dQAFQlA8mQtQRFE7MMsBMYoxHqGAKk7Wx+jkfA3Q9fhvCdDx0lxBw1m5aBGUIz1OwoZprW8PMCDXfAAyMdI4H8MDYwvcUIN1cUbho44dlwF18AANW5bikApUxRPCATv1qTDsK7FYe3lPstJ+YcdJ9cc4BUGI6ksXk-1QwcbTKO1hVxD0IG0XM7JdbDhXPPjTCE4TYXUywmhwWgByMl4zIwyyGGstVbMAhyQHwJyLVc197S81TNSTFNILTJEM0fbMXxAN8qw-EBVJLcsoPSOzUG5KCIAAawQECqvAjqkgACW3V5BFeCUSERV4GKYuiURDG0Wh8VAEG0IwAE98DRAAjOAjHkfwUk27woAgPYPVeG1WuqeRFvQVaICMAjUCMG79vGA81RIZbO3cEgEXCfA-QRMBclwbaAHU4AAchGpl0iJJgMSk91bqIo6KUgCZjlgiY5z+gGcE1FG4FOQEIEwfB1v+kN+teZr9qtD04IDPQSFWtC4JiR1YF4vKEz85tuf85SPGOJq8hurYobVRaFM1fmuopQLyE8zmxTC0ddNMsDaTHX5wpWEylHV8lzMw3lavqxqWoQBh9aSFKXN5G1hdFoklstxXaRt7RkhwDx12AABtPh-gUzXh38Mg6wUX3FwAXWUKO9ddkUEiShh4+Euk5awjt9JHAd+1VkztYMicUSi8hjbq3AGqMJrTsczdbdfC6rqMYV0bgTHUH+uyRR83nBJl2FpYAWWWgAhbl3jAOBJaUuoxP1gLS8zvIrezwu1YTpJg-gdfddXw3opAE3K7N2uQFX93eVWieoCngB5CgXbTt3649r2fd97eeLD6d5EjqO47628orfifEJTHnEvNfANcEAai1BAviGdgroH1srXO+sD5lwYMfBm1dzZRHPgnS+DALKxAoNuJ+z81obQ-tg7wyRtC6GuI4EAawCToGnMxfAWx5oMEAYrYBzYrzJhvHlO8hUszPlzPmcqBCOo-mOH0Ew4gJL2TgO1PifcxQND9NyJa1pYKKEwQ6BCihtBZxgItTA1wEb0UYe6KM3IS6xFQBMMositr60xAdX01QkZ4wMa8G4fpvYkw2paN4jwYgjWdJ8f4JNFrwEUDafAABVKCDQrAAA4mR1BCUhD4gSriIk7q8cxeRcm3HID9RQ7ptzDk+OiN4K0DwwA5mnUBICtEiTnipHYehvpShntLBeGdNJoNVnnYyWsVbTOHJFUoh8lE9FUUBOAddnJpVKnk8IUphQ2lSTACgOTBH+V8vI7pdJpb9JCfgUswzekC1GUvcZqt15TJeAXWZnzfgLIslgkAyyVF2TWRs1KbkBkeFLPsiAhzjk9xAecvi1yQloSyHAEpU8vQdgedQFSiCXlZwmT6D5OsZlfz+UbBgQLVmJWSq-XMqK4BZCyJlW8wiLzZWvKCfYHAlCsFYSAImHjkCgCdNQQQAAFJ0sDkAgGGhAVxgrVr1FajALYmAyAhOQHUbi+QQAShgOgIwQNYJJM1VPHhLiqwlMsXKsAGJBV+jzHgSV9l0AkGQFMTaqwQAACt5DiFHqq+0WxnwABk-RwC9dyPM+qA0qKZQARXSD4mNPr9WapwC6uV191pGEFYCP0MAgauI9MgLJi5M3UDzEDMomA5V7hdSNQVABHVN8A3VE3kHK9EVgBAuhdIKpg7bUBMDdUBD16a42lTgOgVAOrbi+rDOEEwKa01IG9TOvIq1S3aHLUgUs+qLJ3RCdIKdKB9oCEFeTKCjMe2btjb6xE3FMZyC2GAPQmAYDJA+DsRaJhp2mSUEAA)